src/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
```

Comment:
update title

Reply:
what you actually mean , to name somehow else the title ? If that's correct what name you wanna me to enter to title